vision/system-test/quickstart.test.js

```diff
@@ -24,6 +24,6 @@ describe('quickstart', () => {
   it('should detect labels in a remote file', async () => {
     const stdout = execSync('node quickstart.js');
     assert.match(stdout, /Labels:/);
-    assert.match(stdout, /cat/);
+    assert.match(stdout.toLowerCase(), /cat/i);
```

Comment:
don't need both `.toLowerCase()` and `/i` (presuming that's a case-insensitive regex `i`)
```suggestion
    assert.match(stdout, /cat/i);
```